modules/rosa-create-objects.adoc

```diff
@@ -273,6 +273,79 @@ Create a cluster with autoscaling enabled on the default worker machine pool.
 $ rosa create cluster --cluster-name=mycluster -region=us-east-1 --enable-autoscaling --min-replicas=2 --max-replicas=5
 ----
 
+[id="rosa-create-external-auth-provider_{context}"]
+== create external-auth-provider
+
+Add an external authentication provider instead of an internal OIDC provider to approve users to access your cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa create external-auth-provider --cluster=<cluster_name> | <cluster_id> [arguments]
+----
+
+.Arguments
+[cols="30,70"]
+|===
+|Option |Definition
+
+|--claim-mapping-groups-claim <string>     
+|Required. Describes rules on how to transform information from an ID token into a cluster identity.
```

Comment:
Perhaps worth to align with the wording on the root API:
https://github.com/openshift/api/blob/master/config/v1/types_authentication.go#L266

Reply:
The wording is kind of confusing. The best I can interpret is that it would be something like this:

"The reference claim uses an array of string values to construct the groups for the cluster identity."

Reply:
@EricPonvelle I could help provide some context here. Yes, this field is required in rosa cli.

Reply:
Yes, in ROSA CLI flow, the `claim-mapping-groups-claim` is required.

```
rosa create external-auth-provider \
--cluster=lponce-local-01 \
--name=m-entra-id \
--issuer-url=https://login.microsoftonline.com/fa5d3dd8-b8ec-4407-a55c-ced639f1c8c5/v2.0 \
--issuer-audiences=a9464024-b142-4bdf-86c0-a153109cdb14,8a769b34-13c9-4f5b-9933-ec439700ec67 \
--claim-mapping-username-claim=email \
--claim-mapping-groups-claim=groups \
--console-client-id=8a769b34-13c9-4f5b-9933-ec439700ec67 \
--console-client-secret=<SECRET>
```